src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -108,7 +108,6 @@ private MobileAdsDataProvider() {
             "MOBILE_TOP_LEADERBOARD",
             Collections.emptyList(),
             Arrays.asList(
-                "\"rpfl_7450\":[\"15_tier",
```

Comment:
FYI (anyone who's reading it and wonder what `15` is - not author of the PR): https://wikia-inc.atlassian.net/wiki/display/ADEN/Rubicon+Fastlane check the "Slot size mapping" table.